apps/stata/stata.talon

```diff
@@ -0,0 +1,7 @@
+# Commands for the Stata Do-File Editor
+os: windows
+app: stata
+-
+settings():
+    key_wait = 4
```

Comment:
what's the deal with these delays out of curiosity? is the Stata editor particularly slow on windows? I think it's ok unless someone complains eventually, but since there is another issue in this file I figured I'd ask.

Reply:
You are right, these do not seem to be strictly necessary for the pull request. I believe I added them because I have some extra commands for stata that may run into trouble otherwise. Since these are not part of the pull request, there  is no need for the file at all.

 I would suggest to keep calling the other file "stata_do_file_editor" because those commands target the "Do-File Editor" of Stata's IDE, not the command window/terminal. The Stata IDE is similar to R Studio or Spyder for python.